src/cookbook/networking/fetch-data.md

```diff
@@ -35,14 +35,24 @@ Import the http package.
 import 'package:http/http.dart' as http;
 ```
 
-Additionally, in your AndroidManifest.xml file, 
+If you are deploying to Android, edit your `AndroidManifest.xml` file to 
 add the Internet permission.
 
 ```xml
 <!-- Required to fetch data from the internet. -->
 <uses-permission android:name="android.permission.INTERNET" />
 ```
 
+Likewise, if you are deploying to macOS, edit your 
+`macos/Runner/DebugProfile.entitlements` and `macos/Runner/Release.entitlements`
```

Comment:
Should we include the path to the files, or just the filenames?

Reply:
I think what you have here with the path makes sense, as it accounts for the common case and still has the filenames.